canutils/lely-canopen/Makefile

```diff
@@ -170,25 +170,34 @@ MAINSRC    = $(LELYCANOPEN_SRCDIR)/tools/coctl.c
 endif
 
 # Download and unpack tarball if no git repo found
-ifeq ($(wildcard $(LELYCANOPEN_SRCNAME)/.git),)
```

Comment:
ditto

Reply:
@xiaoxiang781216 Here we use curl is a lely-core-master.tar.gz package so it does not contain the .git 

https://gitlab.com/lely_industries/lely-core/-/archive/master/a2ecf70b76a42c25c5338d86e85250bf16fd7023.tar.gz

This I have seen also affects in other places in the nuttx-apps repository 

perhaps due to the copy-paste

should be fixed

Reply:
some examples ifeq ($(wildcard xxxxx/.git),)

apps/audioutils/libmad/Makefile
apps/benchmarks/coremark/Makefile
apps/benchmarks/fio/Makefile
apps/benchmarks/superpi/Makefile

I also believe that it is necessary to create common guidelines to standardize the downloading of external packages in the two NuttX repositories.
Also use, where possible, more generic variable names
example :

Let's use 
LVGL_TARBALL
https://github.com/apache/nuttx-apps/blob/bbcba55bbc357b3c5f651379f55b2917075e4df9/graphics/lvgl/Makefile#L62

for a .zip package

maybe better LVGL_ARCHIVE

Reply:
@simbit18 with this check with `.git` you can keep your custom git repository with your own patches. I think this is a good approach. For this patch I removed it because for some reason it fixed the compilation, but the real problem was somewhere else, so this change is no longer necessary